### PR TITLE
bytesize nf-core modules patch

### PR DESCRIPTION
**Pull request by FranBonath**

### CODE REVIEW

File: markdown/events/2023/bytesize_modules_patch.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Join us for our **weekly series** of short talks: **“nf-core/bytesize”**.
+
+Just **15 minutes** + questions, we will be focussing on topics about using and developing nf-core pipelines.
+These will be recorded and made available at <https://nf-co.re>
+It is our hope that these talks / videos will build an archive of training material that can complement our documentation. Got an idea for a talk? Let us know on the [`#bytesize`](https://nfcore.slack.com/channels/bytesize) Slack channel!
+
+## Bytesize: nf-core modules patch
+
+Did you ever come accross the issue that you need non-nf-core or modified custom modules, but then linting complains when you put them anywhere logical? This is waht `nf-core modules patch` was created for. To hear more, join us for this weeks bytesize talk with Phil Ewels ([@ewels](https://github.com/ewels))!
```
Comment (mashehu) [inline markdown/events/2023/bytesize_modules_patch.md:23]:
```suggestion
Did you ever come accross the issue that you need non-nf-core or modified custom modules, but then linting complains when you put them anywhere logical? This is what `nf-core modules patch` was created for. To hear more, join us for this weeks bytesize talk with Phil Ewels ([@ewels](https://github.com/ewels))!
```

**Review by mashehu — APPROVED** [review]

**Comment by FranBonath** [comment]
I would like an ok from @ewels, also so I know he agrees to give the talk 😉

File: markdown/events/2023/bytesize_modules_patch.md (hunk 1 above)
Comment (ewels) [inline markdown/events/2023/bytesize_modules_patch.md:23]:
```suggestion
Have you ever wanted to tweak an nf-core module for your own use? No need to copy / paste and lose the benefits of linting and updates, instead try using the magic `nf-core modules patch` command to keep track of your modifications.

Sounds too good to be true? Join us to hear more at this week's bytesize talk with Phil Ewels ([@ewels](https://github.com/ewels))!
```

**Review by ewels — APPROVED** [review]
Thanks @FranBonath !

**Pull request merged**